sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/AadCredentialTest.java

```diff
@@ -53,20 +54,37 @@ private void setup(HttpClient httpClient, ConfigurationServiceVersion serviceVer
                 .httpClient(httpClient)
                 .credential(tokenCredential)
                 .endpoint(endpoint)
-                .serviceVersion(serviceVersion);
-
+                .serviceVersion(serviceVersion)
+                .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS));
```

Comment:
I don't think we need to include this anymore. This doesn't configure any allowed headers and query parameters anyways.